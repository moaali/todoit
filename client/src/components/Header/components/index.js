export { default as Logo } from './Logo'
export { default as NavIcon } from './NavIcon'
export { default as HeaderSearch } from './HeaderSearch'
