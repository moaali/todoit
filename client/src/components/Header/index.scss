@import 'settings/index';
@import 'tools/index';
@import 'helpers/index';

.header {
  // Box Model
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 15px;
  position: fixed;
  top: 0;
  left: 0;
  width: 100%;
  z-index: 1000;
  height: 60px;

  @media (min-width: 576px) {
    // Box Model
    padding: 0 30px;
    height: 80px;
  }

  // Cosmetics
  background: white;
  box-shadow: 0 0 3px 3px rgba($color-primary, 0.06);
}
