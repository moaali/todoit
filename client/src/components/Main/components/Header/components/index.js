export { default as FilterCompleted } from './FilterCompleted';
export { default as Title } from './Title';
