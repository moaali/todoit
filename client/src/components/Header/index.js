import React from 'react'
import PropTypes from 'prop-types'

import { Logo, NavIcon, HeaderSearch } from './components';
import './index.scss'

const Header = ({ className, ...restProps }) => (
  <div className={`peers header ${className}`} {...restProps}>
    <div className='peer'>
      <Logo />
    </div>
    <div className='peers peer-greed jc-c'>
      <HeaderSearch />
    </div>
    <div className='peer'>
      <NavIcon />
    </div>
  </div>
);

Header.defaultProps = {
  className: '',
}

Header.propTypes = {
  className: PropTypes.string,
}
export default Header;
