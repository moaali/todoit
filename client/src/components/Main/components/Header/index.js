import React from 'react'

import { FilterCompleted, Title } from './components';

const Header = () => (
  <div className='peers ai-c jc-sb fxw-nw mB-30'>
    <div className="peer">
      <Title />
    </div>
    <div className="peer">
      <FilterCompleted />
    </div>
  </div>
);

export default Header;
