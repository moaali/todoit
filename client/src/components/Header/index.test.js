import React from 'react'

import Header from './'

describe('Header', () => {
  it('should match snapshot', () => {
    const contentElement = shallow(<Header />);
    expect(toJson(contentElement)).toMatchSnapshot();
  })
})
