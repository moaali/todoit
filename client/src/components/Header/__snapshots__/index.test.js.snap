// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match snapshot 1`] = `
<div
  className="peers header "
>
  <div
    className="peer"
  >
    <inject-Logo-with-store />
  </div>
  <div
    className="peers peer-greed jc-c"
  >
    <inject-HeaderSearch-with-store />
  </div>
  <div
    className="peer"
  >
    <NavIcon />
  </div>
</div>
`;
